testutil/rest.go

```diff
@@ -7,6 +7,11 @@ import (
 	"net/http"
 )
 
+type ErrorResponse struct {
```

Comment:
defining a common struct that is returned when your query fails.  tests can just decode to this